api/spec/v1.yaml

```diff
@@ -2149,6 +2187,36 @@ components:
           description: The transaction volume on this day.
           example: 420
 
+    ActiveAccountsList:
+      type: object
+      required: [window_size_seconds, windows]
+      properties:
+        window_size_seconds:
```

Comment:
I like it because it explicitly states the window size of the response windows. This cannot be inferred from the window data itself, because the endpoint can return overlapping windows.